assets/css/default-calendar-grid.css

```
@@ -542,9 +542,83 @@
+.simcal-calendar-actions button.simcal-calendar-action:hover,
+.simcal-calendar-actions button.simcal-calendar-action:focus {
+	background: #f8fafc !important;
+	color: #1e293b !important;
+}
+
+.simcal-calendar-actions .simcal-calendar-action-icon {
+	display: block;
+	flex-shrink: 0;
+	width: 16px;
+	height: 16px;
+	stroke: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)